library/Icingadb/Web/Controller.php

```diff
@@ -188,6 +188,8 @@ function (ViewModeSwitcher $viewModeSwitcher) use (
 
                 try {
                     $preferencesStore = PreferencesStore::create(new ConfigObject([
+                        //TODO: Don't set store key as it will no longer be needed once Web v2.11 is released.
```

Comment:
It's not about when v2.11 is out, but when we don't need to support earlier versions.